src/lib/orm/entities/RpgGuildRankEntity.ts

```diff
@@ -12,7 +12,7 @@ export class RpgGuildRankEntity extends BaseEntity {
 	@Column('varchar', { length: 50 })
 	public name!: string;
 
-	@OneToMany(() => RpgGuildEntity, guild => guild.ranks, { onDelete: 'CASCADE' })
+	@ManyToOne(() => RpgGuildEntity, guild => guild.ranks, { onDelete: 'CASCADE' })
```

Comment:
I don't think the ManyToOne side needs to show the inverse

Reply:
All other references of `@ManyToOne` in the project have an inverse, I'd keep this for consistency.

But if you can prove the behaviour is the same without them, feel free to submit a PR later 👍